documentation/whatsnew/fusetools/fusetools-news-10.3.0.AM2.adoc

```diff
@@ -26,3 +26,12 @@ For example:
 
 When selecting factory method on a Global bean, a lot of possibilities were proposed in the user interface. The list of factory methods for a global bean is now limited to only those methods that match the constraints of the bean's global definition type (bean or bean factory).
 
+== Customize EIP labels in the diagram
+
+When you open the Fuse Tooling preferences page for the editor you will recognize a new list "Preferred Labels" on that page.
+
+image::./images/prefEIPLabels.png[Fuse Tooling editor preference page]
+
+This feature enables you to define the displayed title for each EIP available except endpoints.
```

Comment:
Use this option to define the label of EIP components (except endpoints) shown in the Editor's Design view.